django/test/testcases.py

```diff
@@ -730,6 +732,29 @@ def assertWarnsMessage(self, expected_warning, expected_message, *args, **kwargs
             *args, **kwargs
         )
 
+    # A similar method is available in Python 3.10+.
+    if not PY310:
+        @contextmanager
+        def assertNoLogs(self, logger, level=None):
+            """
+            Assert no messages are logged on the logger, with at least the
+            given level.
```

Comment:
Thanks, I had missed the note about wrapping docstrings are 79 and used 72, in spite of your suggestion being wrapped at 79.